nomad/__init__.py

```diff
@@ -4,15 +4,17 @@
 
 class Nomad(object):
 
-    def __init__(self, host='127.0.0.1', port=4646, timeout=5, region=None, version='v1', verify=False, cert=()):
-        self.host = host
+    def __init__(self, uri='http://127.0.0.1', port=4646, nspace='', token='', timeout=5, region=None, version='v1', verify=False, cert=()):
+        self.uri = uri
         self.port = port
+        self.nspace = nspace
+        self.token = token
         self.timeout = timeout
         self.version = version
         self.verify = verify
         self.cert = cert
 
-        self.requester = api.Requester(host, port, timeout, version, verify, cert)
+        self.requester = api.Requester(uri, port, nspace, token, timeout, version, verify, cert)
```

Comment:
Maybe also add a method within this class to update the requester and re-apply to all the values that are properties, not a necessity but could be a nice feature

Reply:
I make changes! and documented de instantiation of the class... I will push on my next commit to review with you